packages/ddp-client/common/livedata_connection.js

```diff
@@ -568,6 +575,51 @@ export class Connection {
    * @param {Function} [asyncCallback] Optional callback; same semantics as in [`Meteor.call`](#meteor_call).
    */
   apply(name, args, options, callback) {
+    const { stubInvocation, invocation, ...stubOptions } = this._stubCall(name, EJSON.clone(args));
+
+    if (stubOptions.hasStub) {
+      if (!stubOptions.alreadyInSimulation) this._saveOriginals();
+      try {
+        stubOptions.stubReturnValue = DDP._CurrentMethodInvocation
+          .withValue(invocation, stubInvocation);
+      } catch (e) {
+        stubOptions.exception = e;
+      }
+    }
+    return this._apply(name, stubOptions, args, options, callback);
+  }
+
+  /**
+   * @memberOf Meteor
+   * @importFromPackage meteor
+   * @alias Meteor.applyAsync
+   * @summary Invoke a method passing an array of arguments.
+   * @locus Anywhere
+   * @param {String} name Name of method to invoke
+   * @param {EJSONable[]} args Method arguments
+   * @param {Object} [options]
+   * @param {Boolean} options.wait (Client only) If true, don't send this method until all previous method calls have completed, and don't send any subsequent method calls until this one is completed.
+   * @param {Function} options.onResultReceived (Client only) This callback is invoked with the error or result of the method (just like `asyncCallback`) as soon as the error or result is available. The local cache may not yet reflect the writes performed by the method.
+   * @param {Boolean} options.noRetry (Client only) if true, don't send this method again on reload, simply call the callback an error with the error code 'invocation-failed'.
+   * @param {Boolean} options.throwStubExceptions (Client only) If true, exceptions thrown by method stubs will be thrown instead of logged, and the method will not be invoked on the server.
+   * @param {Boolean} options.returnStubValue (Client only) If true then in cases where we would have otherwise discarded the stub's return value and returned undefined, instead we go ahead and return it. Specifically, this is any time other than when (a) we are already inside a stub or (b) we are in Node and no callback was provided. Currently we require this flag to be explicitly passed to reduce the likelihood that stub return values will be confused with server return values; we may improve this in future.
+   * @param {Function} [asyncCallback] Optional callback; same semantics as in [`Meteor.call`](#meteor_call).
+   */
+  async applyAsync(name, args, options, callback) {
```

Comment:
If this method is going to be exposed to the client it needs to be added in the array here ``packages/ddp-client/client/client_convenience.js``